make/manifests.mk

```diff
@@ -7,7 +7,7 @@ OWNER_AND_BRANCH_LOCATION=codeready-toolchain/api/master
 GH_SCRIPTS_URL=https://raw.githubusercontent.com/${OWNER_AND_BRANCH_LOCATION}/scripts
 
 TMP_DIR?=/tmp
-IMAGE_BUILDER?=podman
+IMAGE_BUILDER?=docker
```

Comment:
if we decided to use docker as the default image builder, then we should do it in all repos (and ideally in separate PRs)

Reply:
Sure, let me undo it from this PR.

Reply:
Reverted in 278989c3c667f9ce261e598666d73f167014cf5e